### PR TITLE
GSI GitHub Issue NOAA-EMC/GSI#290.  Port non-operational portions of RadMon to wcoss2.

### PR DESCRIPTION
**Pull request by EdwardSafford-NOAA**
Port non-operational portions of RadMon to wcoss2.  Some general cleanup of non-operational portion of package is included as well.  This completes issue #290

### CODE REVIEW

**Comment by EdwardSafford-NOAA** [comment]
@DavidHuber-NOAA if you have a moment can you review this PR?  Thanks!

**Review by DavidHuber-NOAA — CHANGES_REQUESTED** [review]
Overall, this looks pretty good.  Just a few minor changes.

File: util/Radiance_Monitor/build_RadMon_cmake.sh [hunk 1]
```diff
@@ -23,28 +23,18 @@ echo "top_level = ${top_level}"
...
+if [[ $target = "wcoss_c" || $target = "wcoss_d" ||
+      $target = "orion"   || $target = "wcoss2" ]] ; then
     . $MODULESHOME/init/sh
-    target=wcoss_d
-elif [[ -d /scratch1 ]] ; then
+elif [[ $target = "hera" ]] ; then
     . /apps/lmod/lmod/init/sh
-    target=hera
-elif [[ -d /data ]] ; then
+elif [[ $target = "s4" ]] ; then
```
[the remark this answers is not in the document]
Comment (EdwardSafford-NOAA) [reply util/Radiance_Monitor/build_RadMon_cmake.sh:34]: I'll be glad to do that, but please confirm that $MODULESHOME is defined on s4.  I don't have access to that machine.

File: util/Radiance_Monitor/image_gen/ush/Transfer.sh [hunk 2]
```diff
@@ -58,40 +57,36 @@ echo "SOURCE_PARMS  = $SOURCE_PARMS"
...
    top_parm=${this_dir}/../../parm
-   export RADMON_CONFIG=${RADMON_CONFIG:-${top_parm}/RadMon_config}
 
-   if [[ -s ${RADMON_CONFIG} ]]; then
-      . ${RADMON_CONFIG}
-   else
-      echo "ERROR:  Unable to source ${RADMON_CONFIG}"
-      exit
+   radmon_config=${radmon_config:-${top_parm}/RadMon_config}
+   if [[ ! -e ${radmon_user_settings} ]]; then
```
[the remark this answers is not in the document]
Comment (EdwardSafford-NOAA) [reply util/Radiance_Monitor/image_gen/ush/Transfer.sh:69]: Yes it sure should.

File: util/Radiance_Monitor/build_RadMon_cmake.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (EdwardSafford-NOAA) [reply util/Radiance_Monitor/build_RadMon_cmake.sh:34]: Ok, super!  I'll make that change.

**Comment by EdwardSafford-NOAA** [comment]
@MichaelLueken-NOAA All conversations have been resolved, I've amended the commit, and I think we're good to go now.

**Review by DavidHuber-NOAA — APPROVED** [review]
Looks good!

**Comment by MichaelLueken-NOAA** [comment]
Since @DavidHuber-NOAA has approved these changes, I will give final approval and merge them to the authoritative repository.

**Pull request merged**